mitodl/micromasters
# Added favicon

#### What are the relevant tickets?
Fixes #334 

#### What's this PR do?
Adds a favicon

#### Where should the reviewer start?

#### How should this be manually tested?
Make sure that the favicon shows up when you bookmark the micromasters PR deploy

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?
